cmds/run.go

```diff
@@ -112,15 +112,26 @@ func run() {
 		EnableRbac:        enableRbac,
 	}).Run()
 
+	//// Need to wait for sometime to run another controller.
```

Comment:
ElasticSearch is commented out as it is not compatible with k8sdb/apimachinary master branch.
@tamalsaha

Reply:
cc: @aerokite